src/moa/moa_controllers/moa_controllers/ack_to_motec.py

```diff
@@ -0,0 +1,68 @@
+#/usr/bin/env python3
+import rclpy
+from rclpy.node import Node
+from ctypes import c_uint8
+import time
+
+from ackermann_msgs import AckermannDriveStamped
+from diagnostic_msgs import DiagnosticStatus #maybe change to DiagnosticArray
+from moa_msgs import CANSendReq
+
+class ack_to_mot(Node):
+    def __init__(self):
+        super().__init__('ackermann_to_motec') #node name   
```

Comment:
Also this should really be ackermann_to_can sorry if I keep confusing you when i say ackermann to motec bc that is the general overview but the node its self does not talk to the motec directly but to can dirrectly and so should be called such